### ISSUE TITLE
Requesting the GBIF API with taxonRank

### ISSUE THREAD

**Issue by SophiePamerlon** [gbif/gbif-api#155]
Hello, would it be possible to add taxonRank as a search parameter for the occurrence API? https://techdocs.gbif.org/en/openapi/v1/occurrence
(it'd also be useful on the UI :) )
Thank you!
best,
Sophie

**Comment by gargo-mnhn** [comment]
yes, please ! it is particularly important when we want to have access to species level in dataset predominantly ranking at higher level (iBOL for example)
Best wishes,
Olivier
